homeassistant/components/caldav/config_flow.py

```diff
@@ -0,0 +1,120 @@
+"""Configuration flow for CalDav."""
+
+from collections.abc import Mapping
+import logging
+from typing import Any
+
+import caldav
+from caldav.lib.error import AuthorizationError, DAVError
+import requests
+import voluptuous as vol
+
+from homeassistant import config_entries
+from homeassistant.const import CONF_PASSWORD, CONF_URL, CONF_USERNAME, CONF_VERIFY_SSL
+from homeassistant.data_entry_flow import FlowResult
+from homeassistant.helpers import config_validation as cv
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+STEP_USER_DATA_SCHEMA = vol.Schema(
+    {
+        vol.Required(CONF_URL): str,
+        vol.Required(CONF_USERNAME): cv.string,
+        vol.Optional(CONF_PASSWORD, default=""): cv.string,
+        vol.Optional(CONF_VERIFY_SSL, default=True): cv.boolean,
```

Comment:
`CONF_VERIFY_SSL` doesn't seem used.

Reply:
Thanks, i added support for this everywhere.

I am not sure this is high priority to support in the UI, but figured it wasn't hard to add. Other option is to not support it.

Reply:
It's ok to add it.